src/lighteval/tasks/extended/ifeval/instructions.py

```diff
@@ -21,7 +21,11 @@
 import re
 import string
 
-import langdetect
+from ....utils.imports import is_package_available
```

Comment:
not fond of the ...., would rather have an explicit path (unless there's a reason for this?)